src/frequenz/sdk/timeseries/ev_charger_pool/_ev_charger_pool.py

```diff
@@ -87,7 +87,7 @@ def __init__(
             self._resampler_subscription_sender,
         )
 
-    async def total_current(self) -> FormulaReceiver3Phase:
+    async def current(self) -> FormulaReceiver3Phase:
```

Comment:
What is the reason to rename these functions removing the suffix `total_`, it'd be good if this is stated in the commit description. The documentation for these methods don't need to be updated, right?